docker_run_webapp.sh

```diff
@@ -4,4 +4,6 @@ echo "Starting Docker..."
 docker run --rm \
 	-v $(realpath ~/ncr_model_params):/root/opt/ncr/model_params:ro \
```

Comment:
The next file isn't needed if you do this instead:

```suggestion
	${LOCALMOUNT:+ -v $(realpath ~/ncr_model_params):/root/opt/ncr/model_params:ro} \
```

The above works by only mounting if the `LOCALMOUNT` env is set.

Reply:
Done - this change was implemented and the README.md file was updated accordingly :heavy_check_mark: